Client/Frontend/Browser/Tab.swift

```diff
@@ -187,7 +187,6 @@ class Tab: NSObject {
             webView.accessibilityLabel = NSLocalizedString("Web content", comment: "Accessibility label for the main web content view")
             webView.allowsBackForwardNavigationGestures = true
             webView.allowsLinkPreview = false
-            webView.backgroundColor = UIColor.lightGray
```

Comment:
This line had no effect. 
WKWebView.isOpaque must be changed from true (default) to false for this property to have an effect.

Reply:
Maybe just set this to `.black` here so that in `setNightMode()` you only need to toggle opacity.